src/components/IfPermission/IfPermission.js

```diff
@@ -4,14 +4,20 @@ import { StripesContext } from '../../StripesContext';
 
 const IfPermission = ({ children, perm }) => (
   <StripesContext.Consumer>
-    {stripes => (
-      stripes.hasPerm(perm) ? children : null
-    )}
+    {stripes => {
+      const hasPermission = stripes.hasPerm(perm);
+
+      if (typeof children === 'function') {
+        return children({ hasPermission });
```

Comment:
We don't need to pass the other props down through here?

Reply:
Which other props? `perm`? Whoever is using this form will have access to everything they would have passed to the children, since they'll be rendering the children in the same spot they normally would've.

e.g,
```
renderContacts = () => (
  <IfPermission perm="view-user">
    {({ hasPermission }) => hasPermission ? 
      <Pluggable type="find-user" onSelectUser={this.props.onAddContact} />
      : 'You do not have the ability to add internal contacts.'
    }
  </IfPermission>
```